README.md

```diff
@@ -1,2 +1,61 @@
 # Hello_Microverse_Project
 In this project, I will set up a Hello World using HTML and CSS linters
+# Hello-Microverse
+
+
+> My First Microverse Project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo 
+
+[Coming Soon]
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository: go to Your Cli and run "git clone <repository url>"
+
+- change into the repository:  run 'cd <repository name>'
+
+- Install all the project dependences: Run 'npm Insttall' in your cli
+
+- now you can view the project in the browser using any web server of choice
+
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Eugeneiyukhun)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/eugene-iyukhun-bb491612a)
+- Twitter: [twitter](https://twitter.com/EIyukhun)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse and anyone whose code was used
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You have done great so far, but there are still some changes you need to make before moving to the next milestone. :+1: 
- [ ] As the previous reviewer suggested, you need to use the Microverse readme template, you do not need to remove any section, all you are required to do is to remove the placeholder comments and then customize each section to match your project. This is an example of how a [professional-looking readme should be ](https://github.com/V-Blaze/news-hub/blob/dev/README.md). I'll suggest you look into this.
- [ ] The License link in your Readem leads to a 404 error page. I'll suggest you add the appropriate link to the license file.
- - Screenshot:
![Screenshot from 2023-07-04 15-11-00](https://github.com/Eugeneiyukhun/Hello_Microverse_Project/assets/107632780/d494201a-9ded-4900-889d-71b2a2849577)